bin/crew

```diff
@@ -115,15 +115,34 @@ at_exit do
   ExitMessage.print
 end
 
+def crewlog(str)
+  caller_locations(1, 1).first.tap {|loc| puts "#{loc.path}:#{loc.lineno}:#{caller_locations(3).first.label}:#{str}".lightpurple if @opt_verbose}
+end
+
 def load_json
   # load_json(): (re)load device.json
   json_path = File.join(CREW_CONFIG_PATH, 'device.json')
   @device   = JSON.load_file(json_path, symbolize_names: true)
+  @device_json = JSON.load_file(json_path)
```

Comment:
I sure would like to keep this simple.  Why load twice?

Reply:
Because there is different behavior whether `symbolize_names: true` is set or not. That's the prior behavior, and @Zopolis4 has been doing stuff that works without it being set, so I kept both until we figure out if we actually do need `symbolize_names: true` set everywhere...

Reply:
We don't need it set at all-- it's just a question of the way we access values (i.e. :name vs 'name')

Reply:
What is the default for symbolize_names?  If we don't need to set it, let's not set it.  I'd be more inclined to want it to be flexible for symbols or names if that is possible.

Reply:
It is not set by default (neither is our custom method to symbolize values), and no method for supporting both is available. 

Feel free to change it and all the code that relies on it, or I'll eventually refactor all of the code.

Reply:
Looks like just flipping it to not set it breaks things:
```
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ crew upgrade
/usr/local/bin/crew:394:in `upgrade': undefined method `each' for nil (NoMethodError)

    @device[:installed_packages].each do |pkg|
                                ^^^^^
        from /usr/local/bin/crew:1878:in `upgrade_command'
        from /usr/local/bin/crew:1908:in `<main>'
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $
```

Reply:
uh... well, maybe I can fix that...

Reply:
by and large you'll have to replace :foo with 'foo' whenever json is concerned.

Reply:
not easily... I think this should be left for a cleanup in a future PR...

Reply:
@Zopolis4 Going in reverse, code like `@device[:installed_packages].delete_if { |entry| entry['name'] == @pkg.name }` doesn't work. Any idea how to fix that? (That's only used twice, so it might be easier to take care of those...)

Reply:
I think I got it working...

Reply:
Ok I removed it...